libs/cmap/cmap.go

```diff
@@ -22,31 +22,28 @@ func (cm *CMap) Set(key string, value interface{}) {
 	cm.l.Unlock()
 }
 
-// GetOrSet returns the existing value if present. Othewise, it stores `newValue` and returns it.
-func (cm *CMap) GetOrSet(key string, newValue interface{}) (value interface{}, alreadyExists bool) {
-
+// GetOrSet returns the existing value if present. Otherwise, it stores `newValue` and returns it.
```

Comment:
```suggestion
// GetOrSet returns the existing value if present. Otherwise, it stores `newValue` and returns it. The loaded result is true if the value was loaded, false if stored.
```